registry/vk.xml

```diff
@@ -18536,7 +18692,7 @@ typedef void* <name>MTLSharedEvent_id</name>;
                 <!-- VkPipelineStageFlagBits bitpos="28" is reserved by this extension, but not used -->
                 <enum bitpos="28" extends="VkPipelineStageFlagBits2"        name="VK_PIPELINE_STAGE_2_ACCELERATION_STRUCTURE_COPY_BIT_KHR"/>
             </require>
-            <require extension="VK_KHR_synchronization2,VK_KHR_ray_tracing_pipeline">
+            <require extension="VK_KHR_synchronization2+VK_KHR_ray_tracing_pipeline">
```

Comment:
Is the ``+`` the same as the ``,`` ?

Reply:
found: https://registry.khronos.org/vulkan/specs/1.3/registry.html#_attributes_of_require_tags